src/bundle/Resources/views/admin/search/search_form.html.twig

```diff
@@ -48,10 +48,10 @@
                     data-clear-btn-selector="#{{form.subtree.vars.id}}-clear-btn"
                     data-udw-config="{{ ez_udw_config('single_container', {}) }}"
                     >{{'search.select_content'|trans|desc('Select Content')}}</button>
-                <div class="ez-filters__subtree" 
+                <div class="ez-tag" 
```

Comment:
You should reuse the `src/bundle/Resources/views/Limitation/udw_limitation_value_tag.html.twig` template here.

Reply:
I moved _tag_ template to `views/parts/tag.twig`. I was thinking about putting it in some folder - maybe `views/parts/components/tag.twig`, `views/parts/js_components/tag.twig`etc. but I wasn't sure if it will be better.